solr/core/src/test/org/apache/solr/handler/ThrowErrorOnInitRequestHandler.java

```diff
@@ -1,52 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.solr.handler;
-
-import java.io.IOException;
-import org.apache.solr.common.util.NamedList;
-import org.apache.solr.request.SolrQueryRequest;
-import org.apache.solr.response.SolrQueryResponse;
-import org.apache.solr.security.AuthorizationContext;
-
-/** throws a {@link java.lang.Error} on init for testing purposes */
-public class ThrowErrorOnInitRequestHandler extends RequestHandlerBase {
```

Comment:
Looks to be used in tests: https://github.com/apache/solr/search?q=ThrowErrorOnInitRequestHandler

Reply:
nice find...  i thought maybe this solrconfig was also a dead code path, but appears not

Reply:
restored!